deploy/cicd.rst

```diff
@@ -0,0 +1,52 @@
+Continuous Integration/Deployment
+=================================
+
+Your first choice for CI/CD should be Github Actions. If you are having trouble getting this to work, please ask in #engineering.
+
+.. sidebar:: Heroku Deploys
+
+   If your application is being deployed to Heroku, you should just be using Heroku Pipelines, not Github Actions.
```

Comment:
I'll add a link here later or some additional words, but for now this seems fine.